### DISCUSSION TITLE
Kontenrahmen SKR 42

### DISCUSSION THREAD

**Discussion by JohannMaierhofer** [openjverein/jverein#297] [Ideas]
Generell kann ein Verein seine Konten (Buchungsarten) frei wählen, er muss nur die steuerliche Zuordnung beachten (Buchungsklassen in Jverein).
Es gibt aber auch standardisierte Kontenrahmen wie den SKR 49 der bisher für Vereine gedacht war. Dieser ließ sich in Jverein abbilden.
In Zukunft soll hier der SKR 42 verwendet werden. Dieser lässt sich aber nicht mehr in Jverein abbilden.
Der Unterschied zum SKR 49 ist wie folgt.
In SKR 49 werden Konten fest den Steuerbereichen zugeordnet. Hat man z.B. Anwaltskosten im ideellen Bereich und im Zweckbetrieb so muss man zwei Konten erzeugen. In Jverein also zwei Buchungsarten die jeweils den Buchungsklassen ideeller Bereich und Zweckbetrieb zugeordnet werden.
In SKR 42 braucht man das Konto (Buchungsart) nur einmal anzulegen. Es erfolgt keine direkte Zuordnung zur Buchungsklasse, jetzt Kostenstelle genannt. Die Zuordnung erfolgt jetzt im Buchungssatz.
Für Jverein würde das bedeuten, dass man Buchungsarten ohne feste Zuordnung zu einer Buchungsklasse erzeugen kann. In der Buchung müsste dann neben Buchungsart auch das Attribut Buchungsklasse eingeführt werden. Das muss man dann explizit setzen bzw. automatisch wenn der Buchungsart eine Buchungsklasse zugeordnet ist. Bei Auswertungen muss man dann die Buchungsklasse aus der Buchung lesen.
Soll Jverein das implementieren oder warten bis ein entsprechender Request von Benutzern kommt?

**Comment by dippeal** [comment]
Buchungsarten mit der Verknüpfung zu einer Buchungsklasse ist bereits implementiert. Oder verstehe ich die Frage falsch?
Wir nutzen den SKR49 - kleiner Ausschnitt als Bsp.:
![image](https://github.com/user-attachments/assets/da30f0ff-35ba-4c00-91f0-c63ae2cbd1a4)

**Comment by JohannMaierhofer** [comment]
Bei SKR 42 gibt es keine feste Zuordnung von Buchungsart zur Buchungsklasse. Es wird die Buchung der Buchungsklasse zugeordnet und nicht die Buchungsart. 

Ich habe unter https://www.vibss.de/vereinsmanagement/steuern/buchfuehrung-im-sportverein/kontenplan-und-gewinnermittlung/der-standardkontenrahmen-42-skr-42
folgendes gelesen:

Der DATEV-Kontenrahmen SKR 49 wird ab 2025 ersetzt.

Bislang haben Vereine, die eine Buchhaltungssoftware benutzen, den [DATEV-Kontenrahmen für Vereine, Stiftungen, gGmbHs, (SKR 49)](https://www.vibss.de/vereinsmanagement/steuern/buchfuehrung-im-sportverein/kontenplan-und-gewinnermittlung/der-standard-kontenrahmen-49-skr-49) genutzt. Dieser wird ab 01.01.2025 durch den neuen Kontenrahmen für Vereine - den [SKR42 für Vereine, Stiftungen, gGmbHs von der DATEV](https://www.datev.de/web/de/datev-shop/material/12902-datev-kontenrahmen-skr-42-vereine-stiftungen-ggmbh-4-abs-3-estg/) abgelöst. Ab 2025 wird die DATEV dann nur noch den SKR 42 pflegen.

Hinweise für die Vereine 

Generell ist der Umstieg auf den SKR42 nur für diejenigen Vereine interessant, die aktuell den SKR49 einsetzen und deren Jahresabschlüsse, Körperschaftsteuererklärungen, Anlage Gemeinnützigkeit usw. durch Steuerberater übernommen werden. Im SKR42 sind die Kontonummern 5-stellig, im SKR49 4-stellig.

Wenn also jemand den SKR 42 verwenden will gilt das was ich oben beschrieben habe. In SKR 42 sind die Buchungsarten nicht mehr fest den Buchungsgruppen (Sphären) zugeordnet. Das vermeidet doppelte Buchungsarten. Nun wird die steuerlich relevante Buchungsgruppe (Sphäre) in der Buchung gesetzt. 
Meine Frage war also ob wir die heutige feste Zuordnung der Buchungsart zu Buchungsgruppe aufheben sollen.

Auch wenn man einen eigenen Kontenrahmen verwendet könnte man dadurch doppelte Buchungsarten vermeiden. Erkauft sich das aber mit mehr Konfigurationsaufwand weil man dann den Tätigkeitsbereich in jeder Buchung setzen muss.

**Comment by JohannMaierhofer** [comment]
Man könnte bei den Beitragsgruppen bleiben und den Schalter in den Einstellungen machen ob man eine feste Zuordnung der Beitragsarten zu Beitragsgruppen will oder nicht. Ist sie fest beleibt es für den Benutzer so wie es heute ist. Hebt er sie auf braucht er keine feste Zuordnung mehr machen und muss dann aber die Beitragsgruppe in der Buchung setzen.
Der Buchungsklassensaldo müsste schon aus kompatibilitätsgründen für beide Fälle erst schauen ob in der Buchung eine Beitragsgruppe gesetzt ist und wenn ja diese nehmen. Ist sie nicht gesetzt dann wird sie über die Zuordnung in der Buchungsart genommen.

**Reply by dippeal** [reply]
Den Schalter und die damit verbundene Logik finde ich gut.

**Reply by JohannMaierhofer** [reply]
Ich habe gerade gesehen, dass man auch heute schon Buchungsarten ohne zugeordnete Buchungsklasse erzeugen kann. Man kann auch Buchungen damit machen.
Im Buchungsklassensaldo erscheint dann in der ersten Spalte ein Eintrag "Nicht zugeordnet".
Evtl. braucht man also den Schalter gar nicht und setzt an dieser Stelle an.

**Reply by JohannMaierhofer** [reply]
Ich mache das jetzt mal mit dem Schalter. Es gibt ja schon Anfragen nach SKR 42 Support im JVerein Forum.

**Reply by tolot27** [reply]
Wir wollen als neuer Verein auch gleich damit starten, statt mit SKR 49. Ich finde die Übersicht in dem DATEV-Dokument allerdings ziemlich kompliziert und muss mich noch damit beschäftigen, wie man die "vordefinierten" Konten importiert bekommt. Ein Template gibt es ja dafür noch nicht, oder?

**Reply by JohannMaierhofer** [reply]
Es gibt noch kein Template. Der entsprechende Import/Export aus JVerein ist auch erst verfügbar wenn der Pull Request übernommen wurde.
Ich denke aber, man muss nicht gleich alle Kontenarten (Buchungsarten) anlegen. Es reicht ja auch wenn man erst einmal die erzeugt die man auch bei Buchungen verwenden will. Wenn man eine neue Buchungsart braucht erstellt man sie einfach zusätzlich.

**Reply by JohannMaierhofer** [reply]
Der Pull Request für dieses Feature ist #307

**Reply by JohannMaierhofer** [reply]
Das Feature ist jetzt übernommen. Darum schließe ich die Unterhaltung.
